### DISCUSSION TITLE
ISUCON本番について気になること

### DISCUSSION THREAD

**Discussion by kakeru-one** [Progaku-copy/isucon-knowledge#1] [ISUCON本第１章 輪読会]
# 疑問点
- どんなインフラ構成なんだろう？（越智が知ってはいるので、説明するのでもいいが、不要かも？）
- 本番の時間帯は？
- チームの人数は？

**Comment by kakeru-one** [comment]
「どんなインフラ構成なんだろう？」について
=> 去年はEC2サーバー3台構成だった。初期設定では、EC2サーバー1台ずつそれぞれにAPP/DB/WEBサーバーが全て稼働している状態だった。

**Reply by kakeru-one** [reply]
test

**Reply by Kiyo510** [reply]
## ISUCONのボトルネック要素3点要素
- N+1
- OS、ミドルウェアの設定、Nginxなど
- アルゴリズム
